examples/ColumnGroupsResizeReorderExample.js

```diff
@@ -155,51 +156,58 @@ class ColumnGroupsExample extends React.Component {
         rowsCount={dataList.getSize()}
         width={1000}
         height={500}
-        {...this.props}
-      >
-        {this.state.columnGroupOrder.map(function (columnGroupKey, i) {
-          return (
-            <ColumnGroup
-              key={i}
-              header={
-                <Plugins.ReorderCell
-                  onColumnReorderEnd={onColumnGroupReorderEndCallback}
+        columnsCount={4}
+        getColumnGroup={(i) => {
+          const columnGroupKey = this.state.columnGroupOrder[i];
+          return {
+            key: i,
+            header: (
+              <Plugins.ReorderCell
+                onColumnReorderEnd={onColumnGroupReorderEndCallback}
+              >
+                <Plugins.ResizeCell
+                  onColumnResizeEnd={onColumnGroupResizeEndCallback}
+                >
+                  {columnGroupTitles[columnGroupKey]}
+                </Plugins.ResizeCell>
+              </Plugins.ReorderCell>
+            ),
+            columnKey: columnGroupKey,
+          };
+        }}
+        getColumn={(i) => {
```

Comment:
Just sanity checking: does the logic here ensure that fixed columns come before scrollable columns?

Reply:
Yes, it seems to ensure that.